ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.16.yaml

```diff
@@ -204,6 +204,13 @@ tests:
     - ref: openstack-test-openstack
     workflow: openshift-e2e-openstack-ipi
   timeout: 7h0m0s
+- as: e2e-openstack-ingress-perf
+  minimum_interval: 24h
+  steps:
+    cluster_profile: openstack-vh-mecha-az0
+    env:
+      LOG_LEVEL: debug
```

Comment:
a comment would be nice to explain why we enable debug log level.

Reply:
Oh, I forgot to remove it. Done!